cloudformation/application-setup-basic/stacks/application/create-or-update-application-stack.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+export AWS_PAGER=""
+
+DOCKER_IMAGE_URL=$1
+
+echo "Deploying Docker image $DOCKER_IMAGE_URL ..."
+
+USER_POOL_ID=$(aws cloudformation describe-stacks --stack-name aws101-cognito --output text --query 'Stacks[0].Outputs[?OutputKey==`UserPoolId`].OutputValue')
+USER_POOL_CLIENT_ID=$(aws cloudformation describe-stacks --stack-name aws101-cognito --output text --query 'Stacks[0].Outputs[?OutputKey==`UserPoolClientId`].OutputValue')
+USER_POOL_CLIENT_SECRET=$(aws cognito-idp describe-user-pool-client --user-pool-id $USER_POOL_ID --client-id $USER_POOL_CLIENT_ID --output text --query 'UserPoolClient.ClientSecret')
```

Comment:
I was hoping we could get rid of this, but it's still necessary to pass the client secret into the service stack.